kickoff/jsonexportl10n.py

```diff
@@ -36,7 +36,7 @@ def l10nExport(releaseName):
 def _aggregateBetaLocales(releaseTable, releaseName):
     beta_prefix = releaseName.replace("beta", "b")
 
-    releases = releaseTable.query.filter(releaseTable.name.contains(beta_prefix))
+    releases = releaseTable.query.filter(releaseTable.name.contains(beta_prefix)).filter(releaseTable._shippedAt != None)
```

Comment:
Bah, indeed. I think its 1.0+ https://github.com/mozilla-releng/ship-it/blob/master/requirements/compiled.txt ...